src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -129,6 +129,7 @@ public static class ScriptConstants
         public const string AdminJwtScmValidIssuerFormat = "https://{0}.scm.azurewebsites.net";
         public const string AdminJwtSiteValidIssuerFormat = "https://{0}.azurewebsites.net";
         public const string AdminJwtAppServiceIssuer = "https://appservice.core.azurewebsites.net";
+        public const string AdminJwtAppServiceAudience = "https://appservice.core.azurewebsites.net";
```

Comment:
same comment as the other PR for clarifying incoming vs outgoing auth constants.